server/models/user.js

```diff
@@ -17,11 +17,9 @@ const userSchema = new Schema({
   exploreStandardDev: Number,
 });
 
-userSchema.virtual('articles').get(() => {
+userSchema.virtual('articles').get(function getUserArticles() {
```

Comment:
This was super frustrating - I spent a while trying to figure out why this virtual wasn't working and then I realized it was because 'this' wasn't defined since the callback was an arrow function lol